.travis.yml

```diff
@@ -2,15 +2,14 @@ sudo: false
 language:
   - python
 python:
-  - "3.5"
+  - 3.5
 env:
   - NODE_VERSION="6.9"
 before_install:
   - nvm install $NODE_VERSION
-  - pip install codecov
 install:
   - make deps
```

Comment:
I also noticed that this installs the Python deps, which seems to be unnecessary since all the commands are run in tox anyway, I think the Python deps are being installed twice each time on Travis maybe. But I don't think just removing this will work because it also installs the JavaScript deps which I think are needed. I guess deps needs to be split into separate Python and JavaScript parts in the Makefile.